packages/shared/client/ReplayClient.ts

```diff
@@ -122,15 +129,6 @@ export class ReplayClient implements ReplayClientInterface {
       client.Session.addAnnotationsListener(this.onAnnotations);
     });
   }
-
-  private getSessionIdThrows(): SessionId {
-    const sessionId = this._sessionId;
-    if (sessionId === null) {
-      throw Error("Invalid session");
-    }
-    return sessionId;
-  }
```

Comment:
This method was unsafe. Any caches (like the new recorded events caches) that happened to call an internal method that used it _before_ the session was initialized would fail unnecessarily. (I say "unnecessarily" because those APIs were all async and could have waited or the session to finish initializing.)

Removing it makes the client interface a little less fragile, I think.